deployment/v3/utils/prop_migrator/README.md

```diff
@@ -0,0 +1,225 @@
+| file2.properties   | key2 |
+| ...                | ...  |
+
+### decent_default_value.csv
+
+This file contains information about decent default property values for the latest property files. It has the following structure:
+
+| Property File Name | Key  | Latest Value |
+|--------------------|------|--------------|
+| file1.properties   | key1 | value1       |
+| file2.properties   | key2 | value2       |
+| ...                | ...  | ...          |
+
+### manual-configuration.csv
+
+This file is created or updated by the script and includes unresolved combinations and combinations that require further manual configuration. It has the following structure:
+
+| Property File Name | Key  | Old Value | Latest Value | Configuration Action |
+|--------------------|------|-----------|--------------|---------------------|
+| file1.properties   | key1 | value1    | value2       | update-if-required  |
+| file2.properties   | key2 | value3    | value4       | copy-property       |
+| ...                | ...  | ...       | ...          | ...                 |
+
+## Output
+
+The Property File Updater generates the following output:
+
+### Updated Property Files
+
+This section lists the property files that were updated during the process. It includes the names of the files and the keys that were modified.
+
+### Logs
+
+The script generates a log file (`logs.txt`) that provides detailed information about the update process. It includes information about matching combinations, unresolved combinations, similar latest values, and more. The log file helps you understand the changes made to the property files and allows you to review the update process.
+
+### Manual Configuration CSV
+
+The script creates or updates the `manual-configuration.csv` file. This file includes unresolved combinations and combinations that require further manual configuration. You can use this file to manually update the property files based on the required configuration actions mentioned in the file.
+
+Note:
+
+- It is highly recommended to create backups of your property files before running this script, as it modifies the files in place.
+- The `manual-configuration.csv` file can be used to manually update the property files based on the required configuration actions mentioned in the file.
+
+----
+# Migrator script
+
+
+This Bash script is designed to automate the process of comparing and updating configuration files based on a new set of configurations. It utilizes two Python scripts: `file_comparator.py` and `file_updater.py`.
+
+## Prerequisites
+
+Before running the script, ensure that you have the following:
+
+- Python 3 installed
+- The `file_comparator.py` and `file_updater.py` scripts available in the same directory as this Bash script.
+
+## Usage
+
+1. Clone this repository to your local machine.
+
+2. Open a terminal and navigate to the directory where the script is located.
+
+3. Run the script by executing the following command:
```

Comment:
I feel this readme file has lot of internal details, good to document this, but for a usage this looks very big. can we split this readme file to 2 different files, 
1. specifies about design which give details for the intermediate csv file, knowledge csv files, segregation 2 phthon script etc..
2. specifies about purely usage, how to setup the migrator to run, explains the inputs to be given and explains the output csv and very high level details about what it internally does etc..

Reply:
Updated sir